src/shell/shell.cpp

```diff
@@ -1258,6 +1260,31 @@ void SHELL_Init() {
 	        " Volume in drive %c is %s\n"
 	        " Volume Serial Number is %04X-%04X\n"
 	        "\n");
+	MSG_Add("SHELL_CMD_MOVE_HELP",
+	        "Moves files and renames files and directories\n");
+	MSG_Add("SHELL_CMD_MOVE_HELP_LONG",
+	        "Usage:\n"
+	        "To move one or more files:\n"
+	        "  [color=green]move[reset] [color=cyan][drive:][path]filename1[,...] destination[reset]\n"
+	        "\n"
+	        "To rename a directory:\n"
+	        "  [color=green]move[reset] [color=cyan][drive:][path]dirname1 dirname2[reset]\n"
+	        "Where:\n"
+	        "  [color=white]filename[reset]      Can be either an exact filename or an inexact filename with\n"
+	        "              wildcards, which are the asterisk (*) and the question mark (?).\n"
+	        "              Multiple tokens can be provided and must be seperated by commas.\n"
```

Comment:
"Multiple filenames can be provided"

A regular user wouldn't have a clue what "token" means.

Reply:
Fixed